lib/live_view_native/swiftui/rules_parser.ex

```diff
@@ -1,5 +1,6 @@
 defmodule LiveViewNative.SwiftUI.RulesParser do
   use LiveViewNative.Stylesheet.RulesParser, :mock
```

Comment:
hmmm... I think this is my fault I don't know why I put this in there. I'll fix after this PR is merged